### PR TITLE
Implementation of Pascal Triangle in Java and Python Added (Fixes issue #1818)

### LINKED ISSUE #1818: Implementation of Pascal's Triangle
**This is a(n):**

- [ ] New algorithm
- [x] Update to an existing algorithm with another programming languages
- [ ] Error
- [ ] Proposal to the Repository

**Details:**

I would like to add Pascal's Triangle Algorithm with python and Java. Please assign me for this.

### PR DESCRIPTION
**Pull request by SohelRaja**
Fixes #1818  

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Added the Pascal Triangle implemented using Python and Java to the Pascal_Triangle directory.

#### Languages Used:
- Java
- Python 3

#### Files Added:
- Pascal_Triangle.java
- Pascal_Triangle.py

> We're happy to help you get this ready -- don't be afraid to ask for help.
@SakshiShreya @singh-shreya6 @asha15 @raghav-dalmia @jainaman224 please review the PR.
Thanks!

### CODE REVIEW

File: Pascal_Triangle/Pascal_Triangle.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        // Taking The Input from User
+        System.out.print("Enter the number of rows to be printed: ");
+        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+        // Converting String Into Integer
+        int rows = Integer.parseInt(br.readLine()); 
+        // Calling the Pascal Class
+        new Pascal(rows);
+    }  
+} 
+class Pascal{
```
Comment (jainaman224) [inline Pascal_Triangle/Pascal_Triangle.java]: Add a new line above this line.

Comment (jainaman224) [reply Pascal_Triangle/Pascal_Triangle.java]: Add space before `{`.

File: Pascal_Triangle/Pascal_Triangle.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+                // Assigning the Printable string to string s and printing the s
+                String s = " " + pascal_value + " ";
+                System.out.print(s); 
+                // Calculating the New Pascal Value
+                pascal_value = pascal_value * (row - i) / i;  
+            } 
+            System.out.println(); 
+        } 
+    } 
+}
```
Comment (jainaman224) [inline Pascal_Triangle/Pascal_Triangle.java]: Add sample input and output.

Comment (jainaman224) [reply Pascal_Triangle/Pascal_Triangle.java]: Add a single new line at the end of file.

**Comment by SohelRaja** [comment]
@SakshiShreya @singh-shreya6 @asha15 @raghav-dalmia @jainaman224 Please re-review it. I have resolved conservation.

File: Pascal_Triangle/Pascal_Triangle.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+def Pascal_Triangle(n):  
+    for row in range(1, n + 1):  
+        # Initializing the Initial Value of Pascal Value as 1
+        pascal_value = 1
+        # For printing blank spaces
+        for i in range(1, n - row + 1):
+            print(' ',end=' ')
```
Comment (somya-kapoor) [inline Pascal_Triangle/Pascal_Triangle.py]: Please add space after ,

File: Pascal_Triangle/Pascal_Triangle.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+        for i in range(1, n - row + 1):
+            print(' ',end=' ') 
+        # For calculating the pascal value 
+        for i in range(1, row + 1): 
+            # Creating the printable string and printing the string
+            s = ' ' + str(pascal_value) + ' '
+            print(s, end = " ")  
+            # Calculating the new Pascal Value
+            pascal_value = int(pascal_value * (row - i) / i) 
+        print(end='\n')
```
Comment (somya-kapoor) [inline Pascal_Triangle/Pascal_Triangle.py]: Please add space around the operator

File: Pascal_Triangle/Pascal_Triangle.py (hunk 3 above)
Comment (somya-kapoor) [inline Pascal_Triangle/Pascal_Triangle.py]: Please add space around the operator

File: Pascal_Triangle/Pascal_Triangle.py [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+            s = ' ' + str(pascal_value) + ' '
+            print(s, end = " ")  
+            # Calculating the new Pascal Value
+            pascal_value = int(pascal_value * (row - i) / i) 
+        print(end='\n')  
+
+# Taking the Input from the user  
+rows = int(input("Enter the number of rows to be printed: ")) 
+# Calling the Function
+Pascal_Triangle(rows)
```
Comment (somya-kapoor) [inline Pascal_Triangle/Pascal_Triangle.py]: Please add sample input-output

File: Pascal_Triangle/Pascal_Triangle.py (hunk 4 above)
Comment (SohelRaja) [reply Pascal_Triangle/Pascal_Triangle.py]: I have already done it , just take a look .

Comment (somya-kapoor) [reply Pascal_Triangle/Pascal_Triangle.py]: it should be print(end = '\n') not print(end='\n')

**Comment by somya-kapoor** [comment]
@SohelRaja  After making the changes please squash the commits.

File: Pascal_Triangle/Pascal_Triangle.py (hunk 4 above)
Comment (SohelRaja) [reply Pascal_Triangle/Pascal_Triangle.py]: ok

**Comment by somya-kapoor** [comment]
@SohelRaja Good work. Looks good to me. Just squash the commits, it will be ready for merging.

**Comment by SohelRaja** [comment]
> @SohelRaja Good work. Looks good to me. Just squash the commits, it will be ready for merging.

@somya-kapoor Done!!

**Pull request merged**